Library/Homebrew/cmd/audit.rb

```diff
@@ -512,9 +512,9 @@ def audit_github_repository
     end
 
     problem "GitHub fork (not canonical repository)" if metadata["fork"]
-    if (metadata["forks_count"] < 10) && (metadata["subscribers_count"] < 10) &&
-       (metadata["stargazers_count"] < 20)
-      problem "GitHub repository not notable enough (<10 forks, <10 watchers and <20 stars)"
+    if (metadata["forks_count"] < 50) && (metadata["subscribers_count"] < 50) &&
+       (metadata["stargazers_count"] < 100)
```

Comment:
In my experience the forks count is often 1/10 of the stars count; It’s probably really hard to find a repo on GitHub with ≥50 forks but <100 stars.